packages/web3torrent/src/App.tsx

```diff
@@ -0,0 +1,28 @@
+import { createBrowserHistory } from "history";
+import React from "react";
+import { Route, Router, Switch } from "react-router-dom";
+import "./App.scss";
+import { LayoutFooter, LayoutHeader } from "./components/layout";
+import Welcome from "./pages/welcome/Welcome";
+
+import { RoutePath } from "./routes";
+
+const App: React.FC = () => {
+  const history = createBrowserHistory();
+  return (
+    <Router history={history}>
+      <main className="wrapper__content">
+        <Switch>
+          <Route exact path={`(${RoutePath.Root})`} component={LayoutHeader} />
```

Comment:
Is there a reason for interpolating the string like this instead of assigning the value directly?

Reply:
Zombi code, I'll fix it up